modules/admin_manual/pages/installation/server_prep_ubuntu_18.04.adoc

```diff
@@ -14,7 +14,13 @@
 :transport-huge-pages-url: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/performance_tuning_guide/s-memory-transhuge
 :iscsi_initiator-url: https://help.ubuntu.com/lts/serverguide/iscsi-initiator.html
 
-The target audience for this document is more sophisticated admins with additional needs and setup scenarios.
+== Introduction
+
+The target audience for this document are more sophisticated admins with additional needs and setup scenarios.
+Please consider that Ubuntu 18.04 by design only provides PHP 7.2. Because PHP 7.2 is set end of support by 
+30 Nov 2019, you may want to to consider using PHP 7.3. If you want to install the PHP 7.3 package and other
```

Comment:
PHP 7.2 security support goes to Nov 2020 - so actually there is support for a while yet.

Reply:
rewritten regarding security support